### ISSUE TITLE
Model isn't learning

### ISSUE THREAD

**Issue by sonnguyen129** [irasin/Pytorch_Adain_from_scratch#7]
I trained model for 20 epochs. It seem doesn't learn anything.  Can you help me to fix it?
![1_epoch_2000_iteration](https://user-images.githubusercontent.com/55862188/126373961-773d8892-40e0-4325-840b-45be1335bc2d.png)

**Comment by irasin** [comment]
Hi，@sonnguyentruong129 
I am not sure if this code works well on the newest PyTorch now. However, NST learning is quite sensitive to the learning rate, fusion rate, initial value of weights, or others such as the PyTorch codes themselves.

As my suggestions, you can set the AdaIN fusion rate of contents features and style features to 0 and try to learn the model again. In this case, this model is just learning an AutoEncoder. It should generate results well without 20 epochs.

If the AutoEncoders' learning is ok, you can try using the weights from AutoEncoders as the AdaIN Model, which means setting the fusion rate to the normal values as you want.

Please have a try, Thanks.

**Comment by sonnguyen129** [comment]
Do you mean AdaIN fussion rate  is alpha or lam variable?

**Comment by irasin** [comment]
The fusion rate is alpha, just follow the code `t = alpha * t + (1 - alpha) * content_features`from the model.py.

```

    def generate(self, content_images, style_images, alpha=1.0):
        content_features = self.vgg_encoder(content_images, output_last_feature=True)
        style_features = self.vgg_encoder(style_images, output_last_feature=True)
        t = adain(content_features, style_features)
        t = alpha * t + (1 - alpha) * content_features
        out = self.decoder(t)
        return out

```

**Comment by sonnguyen129** [comment]
Thank you for your reply. Can you explain that use the ```_resize``` function in dataset.py, then using random crop. Why not use torchvision's random crop or resize from the start?

**Comment by irasin** [comment]
Hi, @sonnguyen129 , It is very OK by just using torchvision. However, I didn't  know how to use it when I try to write this repo because I am not familiar with pytorch 3 years ago. That is the reason why I do resize by myself.
